python/sglang/srt/server.py

```diff
@@ -90,9 +90,17 @@
 
 
 @app.get("/health")
-async def health() -> Response:
-    """Health check."""
-    return Response(status_code=200)
+async def health(request: Request) -> Response:
+    """
+    Generate 1 token to verify the health of the inference service.
+    """
+    gri = GenerateReqInput(text="s", sampling_params={"max_new_tokens": 1, "temperature": 0.7})
```

Comment:
Why use temperature 0.7?

Reply:
Just a random number, I want to verify the sampling at the same time.